src/logic/map_objects/tribes/productionsite.cc

```diff
@@ -311,18 +311,20 @@ void ProductionSite::load_finish(EditorGameBase& egbase) {
  * Display whether we're occupied.
  */
 void ProductionSite::update_statistics_string(std::string* s) {
```

Comment:
Nice feature, thanks!
I noticed that it first says "Worker missing" and than changes to "Workers missing" when destroying the road to a mine. But I guess that this is a result of the request system and nothing you can fix here.

Reply:
I have added an additional check for when both request and worker are nullptr.